product_top_category/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+For all categories, compute their top categories. 
```

Comment:
Maybe add that this field is useful for reports and pivot views ?

Reply:
Yes, done